modules/hostap/Kconfig

```diff
@@ -0,0 +1,50 @@
+# WPA Supplicant configuration options
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config WIFI_NM_WPA_SUPPLICANT
+	bool "WPA Suplicant from hostap project [EXPERIMENTAL]"
+	select POSIX_CLOCK
+	select NET_SOCKETS
+	select NET_SOCKETS_PACKET
+	select NET_SOCKETPAIR
+	select NET_L2_WIFI_MGMT
+	select WIFI_NM
+	select EXPERIMENTAL
+	help
+	  WPA supplicant as a network management backend for WIFI_NM.
+
+config WIFI_NM_WPA_SUPPLICANT_THREAD_STACK_SIZE
+	int "Stack size for wpa_supplicant thread"
+	default 8192
+
+# Currently we default POSIX_MAX_FDS to 16 in lib/posix/Kconfig
+# l2_packet - 1
+# ctrl_iface - 2 * socketpairs = 4(local and global)
+# z_wpa_event_sock - 1 socketpair = 2
+# Remaining left for the applications running in default configuration
+
+# Supplicant API is stack heavy (buffers + snprintfs) and control interface
+# uses socketpair which pushes the stack usage causing overflow for 2048 bytes.
+# So we set SYSTEM_WORKQUEUE_STACK_SIZE default to 2560 in kernel/Kconfig
```

Comment:
such a specific number 2560... is there any indication on how much leeway is available with this number?

Reply:
@krish2718 do you know the history behind this value?

Reply:
2048 wasn't working, so, I tried few combinations and IIRC ~2300 should work as well, just to be safe, I added 2048 + 512 = 2560.